### DISCUSSION TITLE
Get CondaHTTPError when installing spacy with conda

### DISCUSSION THREAD

**Discussion by gadgetman4u** [explosion/spaCy#7567] [Help: Installation] [answered]
I am trying to install spacy using conda, and I use this command:

conda install -c conda-forge spacy

as per the spacy.io website installation directions.

However, I keep getting this error:

CondaHTTPError: HTTP 000 CONNECTION FAILED for url <https://conda.anaconda.org/conda-forge/linux-64/current_repodata.json>

Elapsed: -

 

An HTTP error occurred when trying to retrieve this URL.

HTTP errors are often intermittent, and a simple retry will get you on your way.

'https://conda.anaconda.org/conda-forge/linux-64'

How do I fix?  I am running a virtual environment on anaconda and using a company proxy.

Thanks.

**Comment by adrianeboyd** [comment, marked as answer]
Hi, this is probably a question for your company IT folks. Without a proxy this currently works fine.
